src/tests/ftest/datamover/copy_negative.py

```diff
@@ -0,0 +1,368 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+'''
+from data_mover_test_base import DataMoverTestBase
+from os.path import join, sep
+
+class CopyNegativeTest(DataMoverTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test class for Datamover negative testing.
+
+    Test Class Description:
+        Tests the following cases:
+            Bad parameters.
+            Simple error checking.
+    :avocado: recursive
+    """
+
+    # DCP error codes
+    MFU_ERR = "MFU_ERR(-1000)"
+    MFU_ERR_INVAL_ARG = "MFU_ERR(-1001)"
+    MFU_ERR_DCP = "MFU_ERR(-1100)"
+    MFU_ERR_DCP_COPY = "MFU_ERR(-1101)"
+    MFU_ERR_DAOS = "MFU_ERR(-4000)"
+    MFU_ERR_DAOS_INVAL_ARG = "MFU_ERR(-4001)"
+
+    def __init__(self, *args, **kwargs):
+        """Initialize a CopyNegativeTest object."""
+        super(CopyNegativeTest, self).__init__(*args, **kwargs)
+
+    def setUp(self):
+        """Set up each test case."""
+        # Start the servers and agents
+        super(CopyNegativeTest, self).setUp()
+
+        # Get the parameters
+        self.flags_write = self.params.get(
+            "flags_write", "/run/ior/copy_negative/*")
+        self.block_size = self.params.get(
+            "block_size", "/run/ior/*")
```

Comment:
Can be removed and specified directly under /run/ior/* and used as self.ior_cmd.flags.value and self.ior_cmd.block_size.value.

Reply:
Yes, thank you. I was originally using this in a different way along with `block_size_large` and forgot to remove this from the `.py` (`block_size` is in the `.yaml`)